### PR TITLE
pyc8 Audit for DocumentStoreExample

### PR DESCRIPTION
**Pull request by koshy-thomas**
Time.sleep addition to allow restqls to initialise
Unique=False when creating hash index(which allows inserting many docs when we don't have the hash index attribute in the docs)
Stream=True to be set to get real-time updates.
Some PEP8 style reformatting.

### CODE REVIEW

File: docs/collections/documents/document-store-example.md [hunk 1]
```diff
@@ -208,12 +231,24 @@ Let's add a `hash_index` called emails to our collection employees. Please refer
...
+COLLECTION_NAME = 'employees'
+FIELDS = ['email', 'name']
+
+print("--- Connecting to C8")
+
+client = C8Client(protocol='https', host=HOST, port=443,
+                          email=EMAIL, password=PASSWORD,
+                          geofabric=GEOFABRIC)
+
+print("Adding Hash Index...")
```
Comment (diana-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
print("Adding hash index...")
```

File: docs/collections/documents/document-store-example.md [hunk 2]
```diff
@@ -371,21 +424,29 @@ The query `FOR employee IN employees RETURN employee` is equivalent to SQL's SEL
 ## Get realtime updates
 
 Example for real-time updates from a collection in fabric:
-
+:::note
+The 'Stream' parameter should be enabled within the collection to get realtime updates
```
Comment (diana-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
Enable the 'Stream' parameter within the collection to get real-time updates.
```

**Review by diana-macrometa — APPROVED** [review]
A couple minor suggestions, then g2g!

**Comment by koshy-thomas** [comment]
> A couple minor suggestions, then g2g!

**Comment by koshy-thomas** [comment]
Suggestions added. Thanks @diana-macrometa!

**Pull request merged**